dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -833,8 +836,9 @@ private Map<String, Object> getContentXML(final Contentlet contentlet,
         final Map<String, Object> m = new HashMap<>();
         final ContentType type = contentlet.getContentType();
 
-        m.putAll(ContentletUtil.getContentPrintableMap(user, contentlet, allCategoriesInfo));
-
+        final boolean doRender = (BaseContentType.WIDGET.equals(type.baseType()) && Boolean.TRUE.toString().equalsIgnoreCase(render));
+        //Render code
+        m.putAll(ContentletUtil.getContentPrintableMap(user, contentlet, allCategoriesInfo, doRender));
         if (BaseContentType.WIDGET.equals(type.baseType()) && Boolean.toString(true)
                 .equalsIgnoreCase(render)) {
```

Comment:
The Boolean.toString(true).equalsIgnoreCase(render) construct although works adds some unecessary complexity here and not best practice.   It is fine to use this 
"true".equalsIgnoreCase(render) as a general rule for this type of logic,  There should not be a concern that it causes multiple instances of "true" as String literals like "true" are automatically interned by the Java compiler. This means they are stored in a pool, and the same literal will not create new objects. For example, every instance of "true" in your code refers to the same memory location in the string pool.

This appears to go against general guidance to use static final String for repeated strings.   This used to be for two reasons, to prevent duplicate objects and also to have one location for the value of a String when the string is not obvious in itself, e.g. a configuration key or an error message that is used in multiple places.    The guidance is that it is fine to use simple strings that are self evident inline without having to reference a static version e.g "true", "false", "error", "success" as there is no performance penalty in modern jvm using a shared object here does not improve clarity.

Reply:
I do not think this requires a change, just adding for understanding of best practices here.